site/src/analytics-addons.ts

```diff
@@ -130,8 +130,8 @@ const ADDONS: {
   a0d7b954_esphome: {
     name: "ESPHome Home Assistant Add-On",
     documentation:
-      "https://github.com/hassio-addons/repository/blob/master/esphome/DOCS.md",
-    icon: "https://raw.githubusercontent.com/hassio-addons/repository/master/esphome/icon.png",
+      "https://github.com/esphome/home-assistant-addon/blob/main/esphome/DOCS.md",
```

Comment:
This is not correct, this is a different addon.
The esphome repo is defined at the bottom of this list.

Reply:
Yeah just saw that. Should this be removed? Icon and link are broken

Reply:
Probably. If its not in that repo anymore, we should not have it here.

Reply:
The add-on has been removed from the community repository (a couple of days ago).